examples/helper/Cargo.toml

```diff
@@ -0,0 +1,12 @@
+[package]
+name = "helper"
+version = "0.1.0"
+authors = ["Clayton Walker <clayton.m.walker@gmail.com>"]
+edition = "2018"
```

Comment:
let's have `publish = false` just to be sure

Reply:
👍